### PR TITLE
Add desktop version media queries

### PR DESCRIPTION
**Pull request by VaneCode**
-Applied media queries to show a desktop version portfolio.
-Added a complete menu in the desktop version.
-Rearranged the elements in the headline to show the h1, p, button in one column next to the image in the desktop version.
-Rearranged the grid in the works section using grid-areas.
-Rearranged the grid in the about section using grid-areas.
-Added a background image to the footer.
-Resized project 2 to be smaller.
-Resized project 3 to be bigger.
-Made project 4 start in the middle of the screen.
-Corrected the tag div with the tag button in the headline section.
-Made the margin in the footer smaller.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,42 +7,65 @@
...
+                <div class="item-2">
+                    <p class="text-paragraph">
+                        Hello! I am a software developer! I can help you build a product,
+                        feature or website. Take a look of my works.
+                        If you like what you see and have a project you need coded,
+                        don’t hesitate and contact me.
+                    </p>
+                </div>
+                <div class="item-3">
+                    <div class="dark_button text-button"> Star collaboration</div>
```
Comment (Kakalanp) [inline index.html]: Make sure that your **button**s are actually `<button>`'s 😉

File: index.html [hunk 2]
```diff
@@ -63,7 +86,7 @@ <h3 class="text_titleH3">Project 1</h3>
             <div class="container-work" id="project2">
                 <div class="img-work img-work-source2"></div>
                 <div class="div-title flex-work-title">
-                    <h3 class="text_titleH3">Project 2</h3>
+                    <h3 class="text-titleH3">Project 2</h3>
                 </div>
                 <div class="flex-ul-work">
                     <div class="ul-work-item">
```
Comment (Kakalanp) [inline index.html]: [OPTIONAL]

Kindly make sure to make this project a little smaller, it should take closer to 25% of the page's width

File: index.html [hunk 3]
```diff
@@ -84,7 +107,7 @@ <h3 class="text_titleH3">Project 2</h3>
             <div class="container-work" id="project3">
                 <div class="img-work img-work-source3"></div>
                 <div class="div-title flex-work-title">
-                    <h3 class="text_titleH3">Project 3</h3>
+                    <h3 class="text-titleH3">Project 3</h3>
                 </div>
                 <div class="flex-ul-work">
                     <div class="ul-work-item">
```
Comment (Kakalanp) [inline index.html]: Not to be repetitive, but kindly make sure this project is closer to 50% of the screen, that way, your project will look closer to the Figma design 😅

File: index.html [hunk 4]
```diff
@@ -105,7 +128,7 @@ <h3 class="text_titleH3">Project 3</h3>
             <div class="container-work" id="project4">
                 <div class="img-work img-work-source4"></div>
                 <div class="div-title flex-work-title">
-                    <h3 class="text_titleH3">Project 4</h3>
+                    <h3 class="text-titleH3">Project 4</h3>
                 </div>
                 <div class="flex-ul-work">
                     <div class="ul-work-item">
```
Comment (Kakalanp) [inline index.html]: This project width is perfect! 😃

But let's try to make it start at the middle of the page 😁

File: style.css [hunk 5]
```diff
@@ -407,24 +430,43 @@ body {
   background-image: url("./images/bg_skills.png");
 }
 
-.flex-form-section {
+/* Footer section */
+
+footer {
+  margin-top: 10%;
```
Comment (Kakalanp) [inline style.css]: Let's try to reduce the margin a little bit to make it look closer to the other sections 👍

File: style.css [hunk 6]
```diff
@@ -487,24 +518,25 @@ textarea:focus {
 }
 
 .form-button {
-  margin: 20px 18px 64px 14px;
+  width: 100%;
+  margin-bottom: 15%;
```
Comment (Kakalanp) [inline style.css]: This `<button>` should be at max 50% of the curent width, kindly fix that 🥳 

<details><summary>Also, to make it centered you probably will need to change the margin a little bit, click here if you need a hint</summary>
    <pre>
      try: `margin: 0 auto 15%;` 😜 
    </pre>
   </details>

File: index.html [hunk 7]
```diff
@@ -105,7 +128,7 @@ <h3 class="text_titleH3">Project 3</h3>
             <div class="container-work" id="project4">
```
Comment (Kakalanp) [inline index.html:128]: [OPTIONAL]

You're missing a background image here: 

- Your project: 
image

- Figma's: 
image

**Review by Kakalanp — CHANGES_REQUESTED** [review]
# Hello there @VaneCode! hi

### Good job so far! :tada: 

There are some issues that you still need to work on to go to the next project but you are almost there! :pinching_hand: 

so close

## Highlights :+1: 
- No linter errors :heavy_check_mark:
- Repo looks professional :briefcase:
- Solid pull request :scroll:
- Amazing UI design :monocle_face: 

## Required Changes ♻️
***Check the comments under the review.***

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (VaneCode) [reply index.html]: Hello @Kakalanp,

I don't understand how I can make it start in the middle, please could you help me?

File: index.html (hunk 3 above)
Comment (VaneCode) [reply index.html]: Hello @Kakalanp,

I'm confusing, could you explain what is the problem with the class text-titleH3?

File: style.css [hunk 8]
```diff
@@ -189,43 +196,44 @@ body {
...
   align-self: auto;
-  width: 320px;
-  height: 186px;
+  height: 18.6rem;
   background-color: #fff;
   background-repeat: no-repeat;
-  background-size: contain;
-  background-position: cover;
+  background-size: cover;
+  background-position: relative;
```
Comment (Kakalanp) [inline style.css]: `relative` is not a value for [`background-position`](https://developer.mozilla.org/es/docs/Web/CSS/background-position) let's try to remove this line or fix it :bulb:

File: style.css [hunk 9]
```diff
@@ -544,3 +573,252 @@ textarea:focus {
...
+    background-image: url("./images/deco-works-2.PNG");
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: relative;
+  }
+
+  #project4 {
+    grid-area: project4;
+    margin-left: 20%;
+  }
```
Comment (Kakalanp) [inline style.css]: Excellent idea to make it start from the center! :tada: 

Let's try to shrink it a little, maybe we could give it a `width: 60%;`?

File: index.html (hunk 3 above)
Comment (Kakalanp) [reply index.html]: I'm sorry I didn't attach an image to guide you, I was talking about the project card in general, but you nailed it! :ok_hand:

**Review by Kakalanp — CHANGES_REQUESTED** [review]
# STATUS: Changes required :exclamation: 

### Hello there @VaneCode! hi

### Great work on making the changes I previously requested! 👏
You've done well implementing some of the requested changes, but there are still two which aren't addressed yet. :memo: 

almost there

## Highlights :+1: 
- Very creative solutions :heavy_check_mark:
- Amazing communication :brain: 
- Admirable perseverance :1st_place_medal: 

 ## Required Changes ♻️
***Check the comments under the review.***

## Optional suggestions

- Maybe you could start adding the links to your social media and to navigate the page :wink: 

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 9 above)
Comment (VaneCode) [reply style.css]: Hello @Kakalanp,
Thanks for your reply, now I have to change project#4 width and delete the property background-position: relative. Please could you tell me if there are no more changes that I have to do to approve?

**Review by algerina — APPROVED** [review]
Hi @VaneCode , 
Good job so far! :fire: 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSEs
 


 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: style.css (hunk 9 above)
Comment (Kakalanp) [reply style.css]: There wasn't :grin:

Congrats! :clinking_glasses:

**Pull request merged**